h2o-algos/src/test/java/hex/gam/GamCVTest.java

```diff
@@ -0,0 +1,93 @@
+package hex.gam;
+
+import hex.Model;
+import hex.glm.GLMModel;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import water.DKV;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Frame;
+import water.runner.CloudSize;
+import water.runner.H2ORunner;
+import water.util.RandomUtils;
+
+import java.util.Random;
+
+@RunWith(H2ORunner.class)
+@CloudSize(1)
+
+public class GamCVTest extends TestUtil {
+  
+// In this test, I will test all the following cross-validation parameters:
+// 1. fold_assignment = random
+// 2. keep_cross_validation_model
+// 3. keep_cross_validation_predictions
+// 4. keep_cross_validation_fold_assignment
+//
+// If we keep the cross-validation models and the fold assignment, then the prediction using the folds and
+// the predictions kept from cross-validation should yield the same result!  
+  @Test
+  public void testCVBinomial() {
+    Frame train;
```

Comment:
Knitpick/codestyle: No need to define al lthe variables like at if scopes are used.

Reply:
I care about your knitpicking.